hexrd/ui/hexrd_config.py

```diff
@@ -888,8 +888,10 @@ def masked_images_dict(self):
         images_dict = self.images_dict
         for det, mask in self.raw_masks_dict.items():
             for name, img in images_dict.items():
+                if np.issubdtype(img.dtype, np.integer):
+                    img = img.astype(float)
```

Comment:
FYI: this currently isn't working, because `img.astype(float)` is creating a new array (not modifying the existing array in the `images_dict`) and creating a new `img` variable from that new array. So the `images_dict` that is returned doesn't contain the modified array.